src/atomate2/vasp/flows/mp.py

```diff
@@ -0,0 +1,184 @@
+"""
+Module defining flows for Materials Project r2SCAN workflows.
+
+Reference: https://doi.org/10.1103/PhysRevMaterials.6.013801
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from typing import TYPE_CHECKING, Sequence
+
+from jobflow import Flow, Maker
+
+from atomate2.vasp.jobs.mp import (
+    MPGGARelaxMaker,
+    MPGGAStaticMaker,
+    MPMetaGGARelaxMaker,
+    MPMetaGGAStaticMaker,
+    MPPreRelaxMaker,
+)
+
+if TYPE_CHECKING:
+    from pathlib import Path
+
+    from jobflow import Job
+    from pymatgen.core.structure import Structure
+
+    from atomate2.vasp.jobs.base import BaseVaspMaker
+
+
+__all__ = ["MPGGARelax", "MPMetaGGARelax"]
+
+
+@dataclass
+class MPGGARelax(Maker):
+    """
+    Maker to perform a VASP GGA relaxation workflow with MP settings.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    initial_relax_maker : .BaseVaspMaker
+        Maker to generate the first relaxation.
+    initial_static_maker : .BaseVaspMaker
+        Maker to generate the static calculation before the relaxation.
+    final_relax_maker : .BaseVaspMaker
+        Maker to generate the second relaxation.
+    """
+
+    name: str = "MP GGA Relax"
+    initial_maker: BaseVaspMaker | None = field(default_factory=MPGGARelaxMaker)
```

Comment:
What do you think about having a more versatile `mp_compat_converter()` function that takes in any existing `atomate2` `Maker` and applies a `config_dict` of the user's choosing which defaults to `src/atomate2/vasp/sets/BaseMPGGASet.yaml`?

Reply:
I think that would be a nice feature. I think I suggested that at some point. Although I think for the canonical MP workflows we should define them explicitly. I image the function you describe (I think it should be a powerup) would be available for other people to convert non-MP workflows into MP compatible ones. If we use it for defining the canonical workflows it will be a bit confusing for users reading the code trying to understand what is going on.